components/apimgt/org.wso2.carbon.apimgt.persistence.mongodb/src/main/java/org/wso2/carbon/apimgt/persistence/mongodb/utils/MongoDBConnectionUtil.java

```diff
@@ -0,0 +1,118 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.wso2.carbon.apimgt.persistence.mongodb.utils;
+
+import com.mongodb.ConnectionString;
+import com.mongodb.MongoClientSettings;
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoDatabase;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.bson.codecs.configuration.CodecRegistry;
+import org.bson.codecs.pojo.ClassModel;
+import org.bson.codecs.pojo.PojoCodecProvider;
+import org.wso2.carbon.apimgt.persistence.internal.ServiceReferenceHolder;
+import org.wso2.carbon.apimgt.persistence.mongodb.dto.MongoDBDevPortalAPI;
+import org.wso2.carbon.apimgt.persistence.mongodb.dto.MongoDBPublisherAPI;
+import org.wso2.carbon.apimgt.persistence.mongodb.dto.APIDocumentation;
+import org.wso2.carbon.apimgt.persistence.dto.CORSConfiguration;
+import org.wso2.carbon.apimgt.persistence.dto.DeploymentEnvironments;
+import org.wso2.carbon.apimgt.persistence.dto.DevPortalAPI;
+import org.wso2.carbon.apimgt.persistence.dto.PublisherAPI;
+
+import static org.bson.codecs.configuration.CodecRegistries.fromProviders;
+import static org.bson.codecs.configuration.CodecRegistries.fromRegistries;
+
+public class MongoDBConnectionUtil {
+
+    private static final Log log = LogFactory.getLog(MongoDBConnectionUtil.class);
+    private static MongoClient mongoClient = null;
+    private static String database = null;
+    private static final String DEFAULT_DATABASE = "APIM_DB";
```

Comment:
This seems to be defined in MongoDBAtlasAPIConnector as well. Shall we define this in a Constants class. eg: MongoDBPersistenceConstants or MongoDBConstants